### ISSUE TITLE
Error in scroll listener

### ISSUE THREAD

**Issue by VytorCalixto** [Pyozer/introduction_screen#13]
When the body content is too extensive and I scroll it, the following error occurs:

`type 'FixedScrollMetrics' is not a subtype of type 'PageMetrics'`

Version: v1.0.7

**Stack trace:**
```
#0      _IntroductionScreenState._onScroll (package:introduction_screen/src/introduction_screen.dart:163)
#1      NotificationListener._dispatch (package:flutter/src/widgets/notification_listener.dart:127)
#2      Notification.visitAncestor (package:flutter/src/widgets/notification_listener.dart:45)
#3      ViewportNotificationMixin.visitAncestor (package:flutter/src/widgets/scroll_notification.dart:31)
#4      Element.visitAncestorElements (package:flutter/src/widgets/framework.dart:3509)
#5      Notification.dispatch (package:flutter/src/widgets/notification_listener.dart:61)
#6      DragScrollActivity.dispatchScrollStartNotification (package:flutter/src/widgets/scroll_activity.dart:436)
#7      ScrollPosition.didStartScroll (package:flutter/src/widgets/scroll_position.dart:658)
#8      ScrollPosition.beginActivity (package:flutter/src/widgets/scroll_position.dart:650)
#9      ScrollPositionWithSingleContext.beginActivity (package:flutter/src/widgets/scroll_position_with_single_context.dart:116)
#10     ScrollPositionWithSingleContext.drag (package:flutter/src/widgets/scroll_position_with_single_context.dart:246)
```

**Comment by kevincherryholme** [comment]
Any solution for this? If not, any alternative package?

**Comment by VytorCalixto** [comment]
@kevincherryholme I'm still using this package since the error doesn't break the widget tree, but it's showing in my error reporting.

**Comment by kevincherryholme** [comment]
@VytorCalixto Right, I decided to continue using it too. I simply wrapped the problem-area in a try-catch without any error handling. This way the error isn't spamming the console :)

**Comment by Pyozer** [comment]
Hi, sorry not to have answered you I am very busy and I forgot your issue :/

I'm not sure to have enough time to check this soon... but if you want you can do a Pull Request and I will try to see it ;)

**Comment by Pyozer** [comment]
Hi, I released v1.0.8 which should fix this problem, just add type checking.
